yarn.lock

```diff
@@ -0,0 +1,805 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
```

Comment:
Would you mind using `npm` instead of `yarn`? I'd prefer to keep the developer setup to a minimum (so people don't have to install yarn).